nvidia_deeplearningexamples_waveglow.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+title: WaveGlow
+summary: WaveGlow model for generating speech from mel spectrograms (generated by Tacotron2)
+category: researchers
+image: nvidia_logo.png
+author: NVIDIA
+tags: [audio]
+github-link: https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/Tacotron2
+featured_image_1: no-image
+featured_image_2: no-image
+---
+
+```python
+import torch
+hub_model = torch.hub.load(github='nvidia/DeepLearningExamples', model='nvidia_waveglow')
+```
+will load the WaveGlow model pre-trained on [LJ Speech dataset](https://keithito.com/LJ-Speech-Dataset/)
+
```

Comment:
same thoughts as above. Add details on what the model takes as input and gives as output.
For example see: https://pytorch.org/hub/facebookresearch_pytorch-gan-zoo_dcgan/

Reply:
changed strategy of this example to go from plain text to sound using the two torch.hub models in series